openwisp_network_topology/admin.py

```diff
@@ -234,7 +241,7 @@ class NodeLinkMixin(MultitenantAdminMixin):
     other extensions to override the admin queryset
     """
 
-    readonly_fields = ['organization']
+    readonly_fields = ['organization', 'gotten_properties']
```

Comment:
why can't we keep the current name `properties`?
If we can't keep it, I would use `parsed_properties` or something similar, to indicate these are the properties that have been parse while reading the network graph data.